src/main/resources/static/js/popup/ConnectBase.js

```diff
@@ -0,0 +1,23 @@
+class ConnectBase {
+  constructor() {
+    this._url = "http://localhost:3000/auth/";
```

Comment:
Java server runs on port 8080, so if you're going to send request to back-end you have to change port.